public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -105,206 +110,214 @@ code-example(language="bash").
-    we won’t initialize the `selectedHero` as we were doing with `hero`.
+    We don't want any of the heroes to be selected before the user picks a hero, so
+    you won't initialize the `selectedHero` as you did with `hero`.
 
-    Now **add an `onSelect` method** that sets the `selectedHero` property to the `hero` the user clicked.
+    Add an `onSelect` method that sets the `selectedHero` property to the `hero` that the user clicks.
   +makeExample('toh-2/ts/app/app.component.ts', 'on-select', 'app.component.ts (onSelect)')
 
   :marked
-    We will be showing the selected hero's details in our template.
-    At the moment, it is still referring to the old `hero` property.
-    Let’s fix the template to bind to the new `selectedHero` property.
+    At the moment, the template still refers to the old `hero` property.
+    To change the template to bind to the new `selectedHero` property, update the code as follows:
 
   +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'selectedHero-details', 'app.component.ts (template excerpt)')
   :marked
     ### Hide the empty detail with ngIf
 
-    When our app loads we see a list of heroes, but a hero is not selected.
+    When the app loads, you see a list of heroes, but a hero is not selected.
     The `selectedHero` is `undefined`.
-    That’s why we'll see the following error in the browser’s console:
+    That's why the following error appears in the browser's console:
 
   code-example(language="html").
     EXCEPTION: TypeError: Cannot read property 'name' of undefined in [null]
 
   :marked
-    Remember that we are displaying `selectedHero.name` in the template.
-    This name property does not exist because `selectedHero` itself is undefined.
+    Remember how you're displaying `selectedHero.name` in the template?
+    This name property doesn't exist because `selectedHero` itself is undefined.
 
-    We'll address this problem by keeping the hero detail out of the DOM until there is a selected hero.
+    To address this issue, you'll keep the hero detail out of the DOM until there is a selected hero.
+    <!-- CF: Will readers know what "DOM" means? -->
 
-    We wrap the HTML hero detail content of our template with a `<div>`.
-    Then we add the `ngIf` built-in directive and set it to the `selectedHero` property of our component.
+    Wrap the HTML hero detail content of your template with a `<div>`.
+    Then add the `ngIf` built-in directive and set it to the `selectedHero` property of your component.
 
   +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'ng-if', 'app.component.ts (ngIf)')
+  // CF: Would it be helpful to note that now the list of names reappears in the browser?
 
   .alert.is-critical
     :marked
       Remember that the leading asterisk (`*`) in front of `ngIf` is
       a critical part of this syntax.
+  //  CF: is a "critical" type note appropriate here? Could I merge this sentence with
+    the next section?
```

Comment:
Hm, not sure. I've seen this same treatment in a couple of places and I think there's a particular reason behind it. I'm going to have to defer to @kwalrath on this one.